.claude/agents/changeset.md

```
@@ -0,0 +1,47 @@
+   - **major** — breaking API changes
+4. Run `pnpm exec changeset` and follow the interactive prompt.
+
+## Changeset message format
+
+- English only
+- Start with a verb: Add / Fix / Update / Remove
+- Be specific about the change
+- No conventional-commit prefixes (`feat:`, `fix:`, etc.)
+
+Example:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)